eodag/plugins/apis/base.py

```diff
@@ -96,8 +96,8 @@ def discover_product_types(self) -> Optional[Dict[str, Any]]:
         return None
 
     def discover_queryables(
-        self, product_type: Optional[str] = None
-    ) -> Optional[Dict[str, Tuple[Annotated[Any, FieldInfo], Any]]]:
+        self, product_type: Optional[str] = None, **kwargs: Optional[Dict[str, Any]]
```

Comment:
- return type should still be Optional[Dict[str, Tuple[Annotated[Any, FieldInfo], Any]]]
- **kwargs type hint should be Any
- parameters description is missing in docstring